app/main/views/metrics.py

```diff
@@ -36,6 +37,15 @@ def get_metrics():
         elif kv['key'] == 'supplier_metrics':
             metrics['supplier_count'] = {"value": kv['data']['total'], "ts": updated_at}
 
+    insight = insight_service.get_insight(now=None)
+    if insight:
+        try:
+            metrics['month_ending'] = {"value": insight['data']['thisMonth']}
+        except KeyError:
+            metrics['month_ending'] = {"value": ""}
+    else:
+        metrics['month_ending'] = {"value": ""}
```

Comment:
This looks good but you could also simplify by initialising `metrics['month_ending']` to reduce repetition.
```suggestion
    metrics['month_ending'] = {"value": ""}
    insight = insight_service.get_insight(now=None)
    if insight:
        try:
            metrics['month_ending'] = {"value": insight['data']['thisMonth']}
        except KeyError:
            pass
```